consumers/src/unit-test/java/uk/gov/moj/sdt/consumers/util/ResponsesSummaryUtilTest.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.moj.sdt.consumers.util;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.moj.sdt.cmc.consumers.converter.XmlToObjectConverter;
+import uk.gov.moj.sdt.cmc.consumers.model.ResponseType;
+import uk.gov.moj.sdt.cmc.consumers.model.claimdefences.ClaimDefencesResult;
+import uk.gov.moj.sdt.cmc.consumers.util.ResponsesSummaryUtil;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Test class for the consumer gateway.
+ *
+ * @author Mark Dathorne
+ */
+@ExtendWith(MockitoExtension.class)
+class ResponseSummaryUtilTest {
+
+    private ResponsesSummaryUtil responsesSummaryUtil;
+    private XmlToObjectConverter xmlToObjectConverter;
+
+    /**
+     * Method to do any pre-test set-up.
+     */
+    @BeforeEach
+    public void setUpLocalTests() {
+        responsesSummaryUtil  = new ResponsesSummaryUtil(xmlToObjectConverter);
+    }
+
+   @Test
+   void convertToMcolResultObjectsIsSuccessful() {
+        List<ClaimDefencesResult> listObjects = createClaimDefencesList();
+        String xml = responsesSummaryUtil.convertToMcolResultsXml(listObjects);
+        assertFalse(xml.isEmpty());
+        assertTrue(xml.contains("mcolDefenceDetail"));
+        assertTrue(xml.contains("defendantId=\"2\""));
+   }
+
+    private List<ClaimDefencesResult> createClaimDefencesList() {
+        ClaimDefencesResult result1 = createClaimDefencesResult("case11", "resp11",
+                LocalDate.of(2021,10,20),
+                LocalDateTime.of(2021,10,21,11,20,11),
+                ResponseType.DE.name(),
+                "defence11");
+        ClaimDefencesResult result2 = createClaimDefencesResult("case12", "resp12",
+                LocalDate.of(2021,10,27),
+                LocalDateTime.of(2021,10,28,6,8,10),
+                ResponseType.AS.name(),
+                "defence12");
+        ClaimDefencesResult result3 = createClaimDefencesResult("case13", "resp13",
+                LocalDate.of(2021,10,29),
```

Comment:
this seems to be repeated couple of times... please try to create a common class